src/adapterService/edgeAdapterService.ts

```diff
@@ -185,18 +218,42 @@ export module EdgeAdapter {
                 // No matching Edge instance
                 ws.close();
             }
-        }
+        }        
 
         private log(message: string): void {
             this.onLogMessage(message);
         }
 
+        private isMessageForNetworkProxy(message: any): boolean{
+            var jsonObject:any;
+            try {
+                 jsonObject = JSON.parse(message);                
+            } catch (SyntaxError) {
+                console.log("Error parsing message: ", message);
```

Comment:
I have renamed the variables to try to let it more descriptive. I hope it is more readable now.